dynamic_rest/prefetch.py

```diff
@@ -205,13 +239,24 @@ def execute(self):
         # TODO: check if queryset already has values() called
         # TODO: use self.fields
         qs = self.queryset._clone()
-        data = list(qs.values())
 
-        self.merge_prefetch(data)
+        has_fastquery = hasattr(self.model, 'use_fastquery')
+        use_models = has_fastquery and self.model.use_fastquery == False
```

Comment:
Would probably abbreviate this to `getattr(self.model, 'use_fastquery', False)`

Reply:
Are you sure? Every model in server.models does not define this variable so would the default be True then instead of False?

Reply:
Should be able to.  The last argument to `getattr` is the default to use if the attribute doesn't exist.  It's basically the same as what you have, or something like `self.model.use_fastquery if hasattr(self.model, 'use_fastquery') else False`